eng/CredScanSuppression.json

```diff
@@ -4,41 +4,40 @@
         {
             "placeholder": [
                 "secret=",
-                "123",
-                "1234567890",
-                "((***Redacted***by***CredScan***))",
-                "proxyPassword",
-                "pass",
-                "dummyPassword",
-                "P@ssw0rd",
-                "12NewPA$$w0rd!",
-                "Samp1eP@ssw0rd",
-                "facebookapplicationsecret",
-                "updatedfacebooksecret",
                 "serverappsecret"
             ],
             "_justification": "Secret used by test code, it is fake."
         },
         {
-            "placeholder": "*sig=sD3fPKLnFKZUjnSV4qA%2FXoJOqsmDfNfxWcZ7kPtLc0I%3D*",
-            "_justification": "Base-64 encoded SHA-256 of a placeholder above."
-        },
-        {
-            "placeholder": "95o6TL9jkIjNr6HurD6Xa+zLQ+PX9/VWR8fI2ofHatbrUb8kRJ75B6enwRU3q1OP8fmjghaoxdqnwhN7m3pZow=",
-            "_justification": "Well-known Account Key"
+            "placeholder": [
+              "username",
+              "-----BEGIN PRIVATE KEY-----",
+              "administratorLoginPassword"
```

Comment:
This doesn't seem like the right thing to suppress. These will suppress true secrets like private keys and usernames from being flagged. Not too sure if administratorLoginPassword is a JSON field returned but if it is this'll result in those being leaked too.